controller/controller_manager.go

```diff
@@ -108,6 +108,7 @@ func StartControllers(logger logrus.FieldLogger, stopCh chan struct{}, controlle
 	kpc := NewKubernetesPodController(logger, ds, scheme, kubeClient, controllerID)
 	kcfmc := NewKubernetesConfigMapController(logger, ds, scheme, kubeClient, controllerID, namespace)
 	ksc := NewKubernetesSecretController(logger, ds, scheme, kubeClient, controllerID, namespace)
+	kpdbc := NewPDBController(logger, ds, kubeClient, controllerID, namespace)
```

Comment:
Can we have the same naming convention? `NewKubernetesPDBController`

Reply:
Thanl you. Fixed